.github/workflows/greeting.yml

```diff
@@ -23,6 +23,8 @@ jobs:
             Currently, I understand the following commands:
             * `/rebase`:            Rebase this PR onto the master branch
             * `/merge`:             Merge this PR into the master branch
+            * `/hold`:              Adds hold label to prevent merging with /merge
+            * `/free-to-merge`:     Removes the hold label to allow merging with /merge
```

Comment:
==> call it "un-hold"?

Reply:
I called it `free-to-merge` to make it clear and long. We can name it `unhold`, no problems